# HG changeset patch
# User Rik <rik@octave.org>
# Date 1427834674 25200
#      Tue Mar 31 13:44:34 2015 -0700
# Node ID e251b75fab55d32accda889c52ee26864567d371
# Parent  0376c12256607013b3847ce3d9b4deeebb9ac2d9
Fix two typos in GUI strings (bug #44681).

* files-dock-widget.cc (files_dock_widget): "directroy" => "directory".

* shortcut-manager.cc (do_init_data): "Ednings" => "Endings"

diff --git a/libgui/src/files-dock-widget.cc b/libgui/src/files-dock-widget.cc
--- a/libgui/src/files-dock-widget.cc
+++ b/libgui/src/files-dock-widget.cc
@@ -103,17 +103,17 @@ files_dock_widget::files_dock_widget (QW
   _sync_browser_directory_action->setToolTip (
     tr ("Go to current Octave directory"));
   _sync_browser_directory_action->setEnabled ("false");
 
   _sync_octave_directory_action
     = new QAction (resource_manager::icon ("go-last"),
                    tr ("Set Octave directory"), _navigation_tool_bar);
   _sync_octave_directory_action->setToolTip (
-    tr ("Set Octave directroy to current browser directory"));
+    tr ("Set Octave directory to current browser directory"));
   _sync_octave_directory_action->setEnabled ("false");
 
   QToolButton * popdown_button = new QToolButton ();
   popdown_button->setToolTip (tr ("Actions on current directory"));
   QMenu * popdown_menu = new QMenu ();
   popdown_menu->addAction (resource_manager::icon ("user-home"),
                            tr ("Show Home Directory"),
                            this, SLOT (popdownmenu_home (bool)));
diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -244,21 +244,21 @@ shortcut_manager::do_init_data ()
         QKeySequence (prefix + Qt::ShiftModifier + Qt::Key_Tab));
 #else
   init (tr ("Indent Selection"), "editor_edit:indent_selection",
         QKeySequence (ctrl + Qt::Key_Tab));
   init (tr ("Unindent Selection"), "editor_edit:unindent_selection",
         QKeySequence (ctrl_shift + Qt::Key_Tab));
 #endif
 
-  init (tr ("Convert Line Ednings to Windows"), "editor_edit:conv_eol_winows",
+  init (tr ("Convert Line Endings to Windows"), "editor_edit:conv_eol_winows",
         QKeySequence ());
-  init (tr ("Convert Line Ednings to Unix"), "editor_edit:conv_eol_unix",
+  init (tr ("Convert Line Endings to Unix"), "editor_edit:conv_eol_unix",
         QKeySequence ());
-  init (tr ("Convert Line Ednings to Mac"), "editor_edit:conv_eol_mac",
+  init (tr ("Convert Line Endings to Mac"), "editor_edit:conv_eol_mac",
         QKeySequence ());
 
   init (tr ("Goto Line"), "editor_edit:goto_line",
         QKeySequence (ctrl + Qt::Key_G));
   init (tr ("Move to Matching Brace"), "editor_edit:move_to_brace",
         QKeySequence (ctrl + Qt::Key_M));
   init (tr ("Select to Matching Brace"), "editor_edit:select_to_brace",
         QKeySequence (ctrl_shift + Qt::Key_M));
